jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/navigation/DesktopStepperAssert.java

```diff
@@ -62,4 +63,33 @@ public DesktopStepperAssert allStepsIncomplete() {
             Matchers.is("has all steps incomplete"));
         return this;
     }
+
+    /**
+     * Waits until step with the given number is completed.
+     *
+     * @param stepNumber step number in this stepper
+     * @return this {@link DesktopStepperAssert} instance
+     */
+    @JDIAction("Assert that step '{0}' in '{name}' is completed")
+    public DesktopStepperAssert stepCompleted(int stepNumber) {
+        SoftAssert.jdiAssert(new Timer(base().getTimeout() * 1000L)
```

Comment:
new Timer(base().getTimeout() * 1000L

please, move the expression to variable

Reply:
Done